src/VideoIO.jl

```diff
@@ -84,24 +100,8 @@ function __init__()
     av_register_all()
 
     if have_avdevice()
-        DEFAULT_CAMERA_OPTIONS["framerate"] = 30
         AVDevice.avdevice_register_all()
         init_camera_devices()
-        if Sys.isapple()
-            # Note: "Integrated" is another possible default value
-            DEFAULT_CAMERA_OPTIONS["pixel_format"] = "uyvy422"
-            DEFAULT_CAMERA_DEVICE[] = isempty(CAMERA_DEVICES) ? "0" : CAMERA_DEVICES[1]
-        end
-        if Sys.iswindows()
-            DEFAULT_CAMERA_DEVICE[] = string(
-                "video=",
-                isempty(CAMERA_DEVICES) ? "0" : CAMERA_DEVICES[1]
-            )
-        end
-        if Sys.islinux()
-            DEFAULT_CAMERA_DEVICE[] = isempty(CAMERA_DEVICES) ? "" : CAMERA_DEVICES[1]
-        end
-        
```

Comment:
Should still call `init_camera_settings()` here, as a user can call `opencamera` without calling `viewcam()` (see #167).